src/utils/calculateClicks.js

```diff
@@ -0,0 +1,17 @@
+import _ from "lodash";
+
+const calculateClicks = (campaignOrChannel, data) => {
+  const receivedData = data;
+  const clicks = _.filter(receivedData, element => {
```

Comment:
use es6 `filter` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter
chain functions: directly `sumBy` the result of `filter`

Reply:
df73dcba84444ea4376252d33043dcbca7551c19